__tests__/app.test.js

```diff
@@ -167,4 +167,26 @@ describe("PATCH: /api/articles/:article_id", () => {
                 })
         })
     })
-})
\ No newline at end of file
+
+})
+
+describe("GET: /api/users", () => {
+    it("Status 200: responds with an array of user objects, each of which have the following properties: 'username', 'name' and 'avatar_url'. ", () => {
+        return request(app)
+            .get('/api/users')
+            .expect(200)
+            .then(( {body} ) => {
+                const { users } = body
+                expect(users).toBeInstanceOf(Array);
+                
```

Comment:
consider testing for expected array length

Reply:
Test criteria added:
expect(users.length).toBe(4)